src/components/bf-button.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <button
+    :autofocus="autofocus"
+    :autocomplete="autocomplete"
+    :disabled="disabled"
+    :form="form"
+    :formaction="formaction"
+    :formenctype="formenctype"
+    :formmethod="formmethod"
+    :formnovalidate="formnovalidate"
+    :formtarget="formtarget"
+    :name="name"
+    :type="type"
+    :value="value"
+    :loading="loading"
+    :ariaPressed="ariaPressed"
+    :ariaExpanded="ariaExpanded"
+  >
+    <slot></slot>
+  </button>
+</template>
+
+<script>
+export default {
+  name: 'bf-button',
+  data: () => ({ }),
+  props: {
+    autofocus: { type: Boolean, required: false },
+    autocomplete: { type: Boolean, required: false },
+    disabled: { type: Boolean, required: false },
+    form: { type: String, required: false },
+    formaction: { type: String, required: false },
+    formenctype: { type: String, required: false },
+    formmethod: { type: String, required: false },
+    formnovalidate: { type: String, required: false },
```

Comment:
formnovalidate is a boolean, default false

Reply:
As these only apply to button type="submit" maybe we should split that into two components